pkg/kev/dev.go

```diff
@@ -143,11 +153,14 @@ func (r *DevRunner) Run() error {
 				kmd.WithStyle(kmd.LogStyle),
 			)
 
+			match := envRe.FindStringSubmatch(ch)
```

Comment:
I like how we're capturing the environment.

Should we also handle what to do in the case of non matches?

Reply:
We technically shouldn’t need to, only the files that are related to tracked environments are watched, so the chance that there won’t be a match is basically non-existent.